README.md

```diff
@@ -101,6 +101,10 @@ The `scripts` folder in this repository contains scripts that interact with the
 
   Checks all free-text fields in G-Cloud services for "bad words" and generates a CSV report of any bad words found.
 
+* `send_dos_opportunities_email.py`
```

Comment:
This script uses underline notation; all other scripts in this repo appear to use dashes. Could we update the script (assuming this isn't just a typo) to conform to that standard, please?